build/deb_rpm/v2-jf/build-scripts/rpm-install.sh

```diff
@@ -1,7 +1,11 @@
-echo "[jfrog-cli]" > jfrog-cli.repo;
-echo "name=jfrog-cli" >> jfrog-cli.repo;
-echo "baseurl=https://releases.jfrog.io/artifactory/jfrog-rpms" >> jfrog-cli.repo;
-echo "enabled=1" >> jfrog-cli.repo;
-echo "gpgcheck=0" >> jfrog-cli.repo;
+#!/bin/bash
+
+{
+  echo "[jfrog-cli]"
+  echo "name=jfrog-cli"
+  echo "baseurl=https://releases.jfrog.io/artifactory/jfrog-rpms"
+  echo "enabled=1"
+  echo "gpgcheck=0"
+} >> jfrog-cli.repo
```

Comment:
I'm don't know this syntax, but why not echoing a single string instead?
You can also echo this string with sudo directly to /etc/yum.repos.d/jfrog-cli.repo.